code/modules/chemistry/Reagents-Medical.dm

```diff
@@ -1557,3 +1557,41 @@ datum
 					L.contract_disease(/datum/ailment/disease/food_poisoning, null, null, 1)
 				..()
 				return
+
+		medical/dimethyl_sulfoxide
+			name = "DMSO"
```

Comment:
I'd use the full name, like we do with `potassium iodide`.

Reply:
Hmm... potassium iodide ~= iodine. Colloquially it is common, but from a chemistry context it seems misleading or incorrect. 

In the spirit of getting people to learn things through SS13, I'm down with this suggestion.